client/state/login/magic-login/actions.js

```diff
@@ -82,7 +82,7 @@ export const fetchMagicLoginRequestEmail = email => dispatch => {
 };
 
 // @TODO should this move to `/state/data-layer`..?
-export const fetchMagicLoginAuthenticate = ( email, token, tt ) => dispatch => {
+export const fetchMagicLoginAuthenticate = ( email, token ) => dispatch => {
```

Comment:
`email` is not required here, only `token`

Reply:
Good point, I've just removed it.